src/views/Timeline.vue

```diff
@@ -407,13 +408,13 @@ export default {
       return icon
     },
     friendlyTime (datetime) {
-      var moment = require('moment')
```

Comment:
aha! this (dynamic) guy was making the issue 👍 
good job, sir

Reply:
Right. Thanks for the review. 
Will commit now and you can pull in the remote ID PR to continue the review there.